src/app/search/autocomplete.test.ts

```diff
@@ -15,6 +15,7 @@ describe('autocompleteTermSuggestions', () => {
     ['(is:blue jun)', 11],
     ['is:bow is:void', 11],
     ['season:>outl', 12],
+    ['not(', 4],
```

Comment:
This isn't a valid search, it's just a regression test for a bug in here I had at one point.